packages/template-retail-react-app/app/ssr.js

```diff
@@ -34,6 +37,21 @@ const options = {
 
 const runtime = getRuntime()
 
+// const clientId = process?.env?.SLAS_PRIVATE_CLIENT_ID
+const clientId = getConfig().app.commerceAPI.parameters.clientId
+const secret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+const encodedSlasCredentials = Buffer.from(`${clientId}:${secret}`).toString(
+    'base64'
+)
```

Comment:
If you aren't using this elsewhere in the code, I would suggest simply moving this to the `onProxyReq` definition, would make the code blurb just a tiny bit more copy/paste-able. What do you think?